app/src/main/java/tech/ula/utils/ValidationUtility.kt

```diff
@@ -82,5 +76,4 @@ class ValidationUtility(systemResources: Resources) {
     }
 }
 
-sealed class CredentialValidation
-data class Credential(val credentialIsValid: Boolean, val errorMessage: String) : CredentialValidation()
\ No newline at end of file
+data class Credential(val credentialIsValid: Boolean, val errorMessageId: Int = 0)
```

Comment:
I still don't love this class name, as I don't think `Credential` really encapsulates the concept of the data it holds.

Reply:
Changed now to CredentialValidationStatus